manifests/h/Heaventools/PEexplorer/1.99.6/Heaventools.PEexplorer.yaml

```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: Heaventools.PEexplorer
+PackageVersion: 1.99.6
+PackageLocale: "en-US"
+Publisher: Heaventools Software
+PackageName: PE Explorer
+License: Copyright © 2000-2021 Heaventools Software.
+PublisherUrl: http://www.heaventools.com/overview.htm
+PrivacyUrl: http://www.heaventools.com/privacy.htm
+ShortDescription: "PE Explorer is the most feature-packed program for inspecting the inner workings of your own software, and more importantly, third party Windows applications and libraries for which you do not have source code."
+Installers:
+- Architecture: "x64"
+  InstallerType: "inno"
```

Comment:
```suggestion
- Architecture: "x86"
  InstallerType: "inno"
  Scope: "machine"
```

Reply:
Do I have add separate Architecture for 64-bit machines?

Reply:
> Do I have to add separate Architecture for 64-bit machines?

The software is x86, therefore the Architecture is only x86, x86 Architecture can be installed on x64 PCs